src/__tests__/index.spec.ts

```diff
@@ -6,13 +6,41 @@
 
 import { createTestModel } from './utils';
 
-import { BifrostModel } from '..';
+import { BifrostModel, BifrostView } from '..';
 
-describe('Example', () => {
-  describe('ExampleModel', () => {
+import BifrostReactWidget from '../components/BifrostReactWidget';
+
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('Widget wrapping classes', () => {
+  describe('BifrostModel', () => {
```

Comment:
Could you explain to me how does this work? Other than this, it looks fantastic! Thank you so much for setting up the testing scripts!